src/interface/api-outbound.yaml

```diff
@@ -1829,6 +1824,21 @@ components:
         - toParticipantId
         - authToken
       additionalProperties: false
+    ConsentRequestsValidateState:
+      title: ConsentRequestsValidateState
+      description: State of POST consent requests validate
+      type: string
```

Comment:
I think there should be the enumeration
- errored
- success
- OTPisValid

Reply:
done.